oem/ibm/libpldmresponder/file_io_type_lid.hpp

```diff
@@ -152,14 +152,15 @@ class LidHandler : public FileHandler
                       << "\n";
             return PLDM_ERROR;
         }
-        close(fd);
-
         rc = transferFileData(lidPath, false, offset, length, address);
```

Comment:
I just noticed that this transferFileData() call doesn't use the file descriptor, so I guess the file descriptor can be removed?

Reply:
Seems this may be for creating the file.